static/bidder-info/mediafuse.yaml

```diff
@@ -0,0 +1,11 @@
+maintainer:
+  email: "paul@mediafuse.com"
```

Comment:
Please use a group email rather than an individual's address. This is a general requirement due to past issues.